CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/AnalyticConfigurationPane.java

```diff
@@ -490,15 +510,29 @@ public static final SelectableAnalyticPlugin lookupSelectablePlugin(final String
         return NAME_TO_SELECTABLE_PLUGIN_MAP.get(selectableAnalyticPluginName);
     }
 
+    private static void setSuppressedFlag(final boolean newValue) {
+        selectionSuppressed = newValue;
+    }
+
     public final class SelectableAnalyticPlugin {
 
         private final CheckBox checkbox;
+        private ListCell<SelectableAnalyticPlugin> parent;
         private final AnalyticPlugin plugin;
         private PluginParameters parameters;
         private PluginParameters updatedParameters;
 
         public SelectableAnalyticPlugin(final AnalyticPlugin plugin) {
             this.checkbox = new CheckBox();
+            // Allows triggering of selection listener when a checkbox is changed
+            this.checkbox.selectedProperty().addListener((observable, oldValue, newValue) -> {
```

Comment:
What is this trying to achieve?

Reply:
See https://github.com/constellation-app/constellation/pull/215#discussion_r372100518

Reply:
Sorry about the duplicate comments, I accidentally left the page while reviewing and didn't realise GitHub must have submitted my comments automatically on exit.